go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/google/go-github/v28
 require (
 	github.com/golang/protobuf v1.2.0 // indirect
 	github.com/google/go-querystring v1.0.0
+	github.com/mattn/go-colorable v0.1.2 // indirect
```

Comment:
Please revert the contents of this file.

Reply:
done